platforms/posix/src/px4_daemon/server.cpp

```diff
@@ -50,6 +50,12 @@
 #include "pxh.h"
 #include "server.h"
 
+// Cygwin handling
+#if !defined(__PX4_CYGWIN)
+	#define CLIENT_SEND_PIPE_OFLAGS O_RDONLY
+#else
+	#define CLIENT_SEND_PIPE_OFLAGS O_RDWR
```

Comment:
I think adding a comment here why this is needed would make sense.

Reply:
I agree a comment is certainly necessary.